Nemo11235/CS160-Final-Project
# final touch

I accidentally merged delivery-2 to the multiplayer branch, and to avoid accidents, I decided to make this branch based on my previous commit. It does the following:
1. Set up the socket.io server;
2. Implemented multiplayer page and connected to the router;
3. Implemented the multiplayer game mode

Test:
1. open two tabs on the browser, both go to http://localhost:3000/race;
2. enter user ID and the same room ID;
3. enter one row on either one tab and click enter;
4. click on the "send" button at the bottom;
5. the word should be displayed on the other side of the other tab.
